### DISCUSSION TITLE
ActiveDots clip behind above area chart when stacking areas

### DISCUSSION THREAD

**Discussion by mjuopperi** [recharts/recharts#4170] [Q&A]
As seen in this picture, the ActiveDots get overlapped by the Area above them. Is there a way to make that not happen? Since it is SVG, z-index does not work.
<img width="498" alt="image" src="https://github.com/recharts/recharts/assets/6764054/32166b7d-714e-422f-8bf0-79a3de00bc67">

You can see the issue here: https://codesandbox.io/p/sandbox/stacked-area-chart-ix341?file=%2Fsrc%2FApp.tsx

**Comment by ckifer** [comment]
I don't think there is currently a way to address this out of the box due to the order of rendering, hopefully we can address it down the road!
